web/src/features/hostdashboard/HostRevenueCards/MonthlyComparison.jsx

```diff
@@ -6,7 +6,7 @@ function MonthlyComparison({ data }) {
     return (
         <div className="mc-comparison-card">
             <BarChart
-                width={1000}
+                width={820}
```

Comment:
Graag geen inline styling 
```suggestion
                width={820}
```

Reply:
Dat zat al zo erin, maar ik zal het even aanpassen